Formula/sc-im.rb

```diff
@@ -0,0 +1,26 @@
+class ScIm < Formula
+  desc "Spreadsheet program for the terminal, using ncurses"
+  homepage "https://github.com/andmarti1424/sc-im"
+  url "https://github.com/andmarti1424/sc-im/archive/v0.6.0.tar.gz"
+  sha256 "5da644d380ab3752de283b83cce18c3ba12b068d0762c44193c34367a0dcbc38"
+  head "https://github.com/andmarti1424/sc-im.git", :branch => "freeze"
+
+  depends_on "ncurses"
+
+  def install
+    ENV.prepend "LDFLAGS", "-L#{Formula["ncurses"].opt_lib}"
```

Comment:
doesn't look like this is actually needed

Reply:
Good point, thanks!

I just checked it; it indeed works for me without that line.